docs/config.md

```diff
@@ -660,28 +660,77 @@ export PROMREGATOR_AUTHENTICATOR_BASIC_PASSWORD=myPassword
 java -Dspring.config.location=file:/path/to/your/myconfig.yaml -jar promregator-0.0.1-SNAPSHOT.jar
 ```
 
-#### Option "promregator.authenticator.oauth2xsuaa.tokenServiceURL" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA)
+#### Option "promregator.authenticator.oauth2xsuaa.tokenServiceURL" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA, *deprecated*)
+Specifies the URL of the OAuth2 endpoint, which contains the token service of your authorization server in case of global authentication. Typically, this is the endpoint with the path `/oauth/token`, as Promregator will try to perform to establish a ["Client Credentials"-based authentication](https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2#grant-type-client-credentials). Deprecated, please use `promregator.authenticator.oauth2xsuaaBasic.tokenServiceURL` instead.
+
+#### Option "promregator.authenticator.oauth2xsuaaBasic.tokenServiceURL" (mandatory, if using promregator.authenticator.type=OAuth2XSUAABasic)
 Specifies the URL of the OAuth2 endpoint, which contains the token service of your authorization server in case of global authentication. Typically, this is the endpoint with the path `/oauth/token`, as Promregator will try to perform to establish a ["Client Credentials"-based authentication](https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2#grant-type-client-credentials).
 
-#### Option "promregator.authenticator.oauth2xsuaa.client_id" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA)
+#### Option "promregator.authenticator.oauth2xsuaaCertificate.tokenServiceCertURL" (mandatory for if using promregator.authenticator.type=OAuth2XSUAACertificate)
```

Comment:
Would be possible to change the naming here to `promregator.authenticator.oauth2xsuaaCertificate.tokenServiceURL` (... no `Cert`).

Then it would be the same like the other  `tokenServiceURL`s. The `Cert` is implied by the prefix.

As always in live there are pros an cons (... this the current approach it is better visible that it is something else).

In case I should change it I can adjust. @eaglerainbow : just let me know ...

Reply:
I get your argument, but given the fact that this typically is also a different server and hence a different URL, I prefer `tokenServiceCertURL` (with Cert).
Perhaps a brief comment along the lines of 

> Note that this attribute is called `tokenServiceCertURL` and not `tokenServiceURL`!

might be helpful, though.

Reply:
done.